pages/index.js

```diff
@@ -1,30 +1,73 @@
-import React from "react";
+import React, { useRef, useState} from 'react';
 
 import TodoInput from "../components/todo-input";
 import TodoList from "../components/todo-list";
 
 import styled from "styled-components";
 
 export default function Home() {
+
+  const [todos, setTodos] = useState([]);
+  const [index, setIndex] = useState(1);
+
+  const addTodo = ({content, date}) => {
+    if ((content === undefined) || (date === undefined)) {
+      alert('모든 항목을 입력해주세요!');
+      return;
+    }
+
+    else if (String(date).length != 8){
+      alert('날짜를 올바른 형식으로 입력해주세요!');
+      return;
+    }
+
+    const todo = {
+      id: index,
+      content,
+      date,
+      checked: false
+    }
+    setTodos(todos.concat(todo)); 
+    setIndex(index + 1);
+    alert('입력 완료!')
+  }
+
+  const todoRemove = (id) => {
+    setTodos(todos.filter(todo => todo.id !== id))
+  }
```

Comment:
```suggestion
  const handleDelete = (index) = () => {
    setTodos([ ...todo.slice(0, index), ...todo.slice(index+1, todo.length)]);
  }
```
1. 함수 네이밍 위와 마찬가지로 ```handleDelete```로 해주시면 좋아요.
2. 함수가 또 다른 화살표 함수를 리턴하게 정의했는데, 이 함수가 ```components/todo-list.js```에서 어떻게 쓰이는지 봐주시면 됩니다.
3. ```todo-list```에서 ```todo-item```을 ```map```하면서 ```index```를 이용할 수 있어 따로 ```index``` state를 선언하지 않고 ```id```가 없어도 삭제할 수 있습니다. 
```index```가 잘 들어왔다고 가정했을 때 ```spread operator```와 ```slice()```를 이용해 ```todo``` 값을 업데이트 할 수 있습니다. [참고](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)